jobs/keepalived/spec

```diff
@@ -0,0 +1,27 @@
+---
+name: keepalived
+
+description: "The keepalived job can be used to add a VRRP IP address to enforce HA on haproxy release"
+
```

Comment:
Maybe we should wait for this feature to be implemented before mentioning it in spec file ?